app/cdap/components/ConnectionList/index.tsx

```diff
@@ -51,9 +52,14 @@ const DatasetWrapper = () => {
   const queryParams = new URLSearchParams(loc.search);
   const pathFromUrl = queryParams.get('path') || '/';
   const [loading, setLoading] = useState(false);
+  const [isErrorOnNoWorkspace, setIsErrorOnNoWorkSpace] = useState(false);
 
-  const toggleLoader = () => {
-    setLoading(!loading);
+  const toggleLoader = (value: boolean, isError?: boolean) => {
+    console.log('value, isError', value, isError);
```

Comment:
remove console.log, check all thru the pr and remove everywhere

Reply:
Done